servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -197,12 +207,58 @@ public synchronized void setPinned(boolean pinned) {
     mPinned = pinned;
   }
 
+  /**
+   * Get the username of the inode
```

Comment:
Tachyon code now tries to apply the following convention to javadoc getters:
`@return the username of the inode`